pydoctor/astutils.py

```diff
@@ -34,3 +35,26 @@ def bind_args(sig: Signature, call: ast.Call) -> BoundArguments:
         if kw.arg is not None
         }
     return sig.bind(*call.args, **kwargs)
+
+
+if sys.version_info[:2] >= (3, 8):
```

Comment:
You could use `literal_eval` to avoid a checking the python version, I think.

Reply:
I think I prefer what I have currently since `ast.literal_eval` can raise so many exceptions. And we don't actually need to evaluate any expressions IMO.